sdk/extensions/sendle/karrio/providers/sendle/tracking.py

```diff
@@ -44,11 +47,11 @@ def _extract_detail(
         tracking_number=tracking_number,
         events=[
             TrackingEvent(
-                date=DF.fdate(event.scan_time, "%Y-%m-%dT%H:%M:%SZ"),
+                date=_best_time(DF.fdate, event),
                 description=event.description,
-                location=event.location,
+                location=f"{event.origin_location} to {event.destination_location}" if (event.origin_location and event.destination_location) else event.location,
```

Comment:
This isn't super nice, but since there's no way to represent a destination location in TrackingEvent currently I figured this at least gives all the information to the consumer.

Reply:
That was clever 👍🏿